pdf2image/pdf2image.py

```diff
@@ -73,12 +73,12 @@ def convert_from_path(pdf_path, dpi=200, output_folder=None, first_page=None, la
     current_page = first_page
     processes = []
     for _ in range(thread_count):
-        # A unique identifier for our files if the directory is not empty
-        uid = str(uuid.uuid4())
+        # Get the filename of the output file.
+        out_file = pdf_path.rpartition('.')[0] if not out_file else out_file
```

Comment:
Line 76 and 77 can be deleted because of `out_file=str(uuid.uuid4())` in function definition.